normandy/settings.py

```diff
@@ -179,6 +180,7 @@ class Base(Core):
     CAN_EDIT_ACTIONS_IN_USE = values.BooleanValue(False)
     ADMIN_ENABLED = values.BooleanValue(True)
     ACTION_IMPLEMENTATION_CACHE_TIME = values.IntegerValue(60 * 60 * 24 * 365)
+    LOGIN_REDIRECT_URL = values.Value('/control/')
```

Comment:
I'm not convinced this needs to be a setting, but if it should be a setting, this is where it should go.